src/redux/books/books.js

```diff
@@ -0,0 +1,39 @@
+import { v4 as uuid } from 'uuid';
+
+const ADD = 'bookstore/books/ADD';
+const DELETE = 'bookstore/books/DELETE';
```

Comment:
[Optional] Don't you think we should use a more descriptive variable name like `ADD_BOOK` and `DELETE_BOOK`? 🤔

Reply:
I thought we won't need that as we are following the ducks pattern `npm-module-or-app/reducer/ACTION_TYPE` would.
I will modify it. Thank you.